capabilityquerier/pkg/capabilityquerier/capabilityquerier.go

```diff
@@ -37,10 +37,7 @@ var tls13 = "TLS 1.3"
 var tlsUnknown = "TLS version unknown"
 var tlsNone = "No TLS"
 
-// Need to define timeout or else it is infinite
-var netClient = &http.Client{
-	Timeout: time.Second * 35,
-}
```

Comment:
It actually is not necessary, since there is already a http client passed in that has the same Timeout property. If you look at the parameters for the GetAndSendCapabilityStatements function, the args map that is passed in contains a http client. If you follow to where this function is getting called in the main file and find where this args map is being populated, you will see that it is populated with a http client with the same exact Timout, and therefore that netClient http client was removed, since there was already an http client being used that did the same thing!

Reply:
^^ bingo! https://github.com/onc-healthit/lantern-back-end/blob/7622e4361218b200554793048bda3742b15c86d3/capabilityquerier/cmd/main.go#L115

Reply:
oh okay, i missed it in the main.go file!